assessment.sql

```diff
@@ -5,24 +5,64 @@ CREATE DATABASE "sql-crud-assessment";
 
 -- Question 1. Create a table called `licensees` to hold data in columns surname,given_name,gender,height,weight,dob,eye_color,hair_color.  See `data/licensees.csv` for example data.  Don't forget that all tables should have an id column as a unique identifier for a row.
 
--- your code here
+CREATE TABLE licensees (
+  id SERIAL PRIMARY KEY
```

Comment:
Damn, that's some rookie spit

On Wed, Nov 4, 2015 at 2:53 PM, Saad Rhoulam <notifications@github.com>
wrote:

> In assessment.sql
> <https://github.com/ga-wdi-boston/sql-crud-assessment/pull/29#discussion_r43930250>
> :
>
> > @@ -5,24 +5,64 @@ CREATE DATABASE "sql-crud-assessment";
> >
> >  -- Question 1. Create a table called `licensees` to hold data in columns surname,given_name,gender,height,weight,dob,eye_color,hair_color.  See `data/licensees.csv` for example data.  Don't forget that all tables should have an id column as a unique identifier for a row.
> >
> > --- your code here
> > +CREATE TABLE licensees (
> > +  id SERIAL PRIMARY KEY
>
> Commas separate column definitions! The comma missing at the end of this
> line will prevent this query from running.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/ga-wdi-boston/sql-crud-assessment/pull/29/files#r43930250>
> .
>



-- 
Regards,
Jeff Meyer

857-206-3572